modules/@apostrophecms/notification/ui/apos/scss/notification.scss

```diff
@@ -0,0 +1,7 @@
+.apos-notifications{
```

Comment:
They are too close together at the moment. In the Figma designs they look to be 8px apart vertically.
![Screen Shot 2020-08-24 at 5 42 00 PM](https://user-images.githubusercontent.com/1155984/91103510-472ae180-e631-11ea-8585-950c25748e0c.png)

![Screen Shot 2020-08-24 at 5 42 34 PM](https://user-images.githubusercontent.com/1155984/91103512-472ae180-e631-11ea-80da-d0d20a3695fe.png)

Reply:
Done